cmd/notify-mailer/main_test.go

```diff
@@ -26,26 +26,36 @@ func TestCheckpointIntervalOK(t *testing.T) {
 		{interval{end: 10}},
 		{interval{start: 10, end: 15}},
 	}
-	for _, testcase := range okCases {
-		err := testcase.testInterval.ok()
-		test.AssertNotError(t, err, "valid interval produced ok() error")
+	for _, tc := range okCases {
+		t.Run(fmt.Sprintf("interval %v", tc.testInterval), func(t *testing.T) {
+			err := tc.testInterval.ok()
+			// NOTE: Do we need subtests in cases where there are only assertions?
```

Comment:
Sorry, I meant to actually try to figure this out and forgot. If the (main) goal of adding subtests is to prevent the tests from stopping after an error, I was debating whether it was easier to just wrap these tests like all the others or to modify the `AssertNotError` defined in `boulder/test/test-tools.go` to not throw a wrench in anything to begin with. I suspect the answer is to leave the helper function alone.